eng/code-quality-reports/src/main/resources/checkstyle/checkstyle-suppressions.xml

```diff
@@ -115,11 +115,14 @@
   <!-- Don't check for JavaDocPackage in samples or tests -->
   <suppress checks="JavadocPackage" files=".*[/\\](samples|test)[/\\].*\.java"/>
 
-  <!-- Suppress lineLength for samples and tests -->
+  <!-- Suppress LineLength for samples and tests -->
   <suppress checks="LineLength" files=".*[/\\](samples|test)[/\\].*\.java"/>
 
-  <!-- Suppress lineLength for Storage on Preview 3. https://github.com/Azure/azure-sdk-for-java/issues/5125 -->
-  <suppress checks="LineLength" files=".*[/\\]com[/\\]azure[/\\]storage[/\\]*"/>
+  <!-- Suppress LineLength for Storage on Preview 3. https://github.com/Azure/azure-sdk-for-java/issues/5125 -->
+  <suppress checks="LineLength" files=".*[/\\]com[/\\]azure[/\\]storage[/\\]"/>
+
+  <!-- Suppress LineLength for Track 1 libraries. -->
+  <suppress checks="LineLength" files=".*[/\\]com[/\\]microsoft[/\\]"/>
```

Comment:
```suggestion
  <suppress checks="LineLength" files=".*[/\\]com[/\\]microsoft[/\\].*"/>
```

Reply:
Not required. The regex will match still.

Reply:
We just want the regex to result in an isMatch when we evaluate a file name with it. The `.*` isn't needed to basically take anything else.

Reply:
To double check, the suppression line here basically does the logic. It looks for any match to that regex, then returns true or false if a match is found.

https://github.com/checkstyle/checkstyle/blob/master/src/main/java/com/puppycrawl/tools/checkstyle/filters/SuppressFilterElement.java#L195